frontend/translations/en.yml

```diff
@@ -324,6 +324,8 @@ en:
     delete_folder: Delete the Folder
     delete_encryptables: Delete the Credentials
     configure: Configure Members
+    encryption_algorithm: "This Team uses "
```

Comment:
Data inside this team is encrypted with ... describe a litte bit more what's happening